ee/federated-saml/api/admin/[id]/index.ts

```diff
@@ -40,6 +41,12 @@ const handleGET = async (req: NextApiRequest, res: NextApiResponse) => {
     const app = await samlFederatedController.app.get(id);
     const metadata = await samlFederatedController.app.getMetadata();
 
+    await sendAudit({
```

Comment:
Do we really need to `await` here and in other places before sending the response back ?

Reply:
Good suggestion, need to think about it.